.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Tests
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Install dependencies
+      run: sudo apt-get -y install python3-pip ninja-build clang-format
+    - name: Install latest Mesone
+      run: pip3 install --user meson
+    - name: Prepare SDK
+      run: wget https://download.panic.com/playdate_sdk/Linux/PlaydateSDK-latest.tar.gz && mkdir ../PDSDK && tar xvf PlaydateSDK-latest.tar.gz --strip-components=1 -C ../PDSDK/
```

Comment:
Why do we need Playdate SDK? All tests are now just pure C++ tests without any dependencies on Playdate.

Reply:
No, I was wrong, having PD SDK makes a lot of sense. Most likely we will test something that just includes "pd_api.h".